src/main/java/bio/terra/cli/command/workspace/ListUsers.java

```diff
@@ -1,32 +1,34 @@
 package bio.terra.cli.command.workspace;
 
-import bio.terra.cli.auth.AuthenticationManager;
-import bio.terra.cli.context.GlobalContext;
-import bio.terra.cli.context.WorkspaceContext;
+import bio.terra.cli.command.helperclasses.BaseCommand;
+import bio.terra.cli.command.helperclasses.FormatOption;
 import bio.terra.cli.service.WorkspaceManager;
 import bio.terra.workspace.model.RoleBinding;
 import bio.terra.workspace.model.RoleBindingList;
-import java.util.concurrent.Callable;
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 
 /** This class corresponds to the third-level "terra workspace list-users" command. */
 @Command(name = "list-users", description = "List the users of the workspace.")
-public class ListUsers implements Callable<Integer> {
+public class ListUsers extends BaseCommand {
 
-  @Override
-  public Integer call() {
-    GlobalContext globalContext = GlobalContext.readFromFile();
-    WorkspaceContext workspaceContext = WorkspaceContext.readFromFile();
+  @CommandLine.Mixin FormatOption formatOption;
 
-    new AuthenticationManager(globalContext, workspaceContext).loginTerraUser();
+  /** List all users of the workspace. */
+  @Override
+  protected void execute() {
     RoleBindingList roleBindings =
         new WorkspaceManager(globalContext, workspaceContext).listUsersOfWorkspace();
-    for (RoleBinding roleBinding : roleBindings) {
-      System.out.println(roleBinding.getRole());
```

Comment:
Looks like `RoleBinding` is an `AutoValue`, does this play nice with Jackson?

Reply:
I'm not sure it's an `AutoValue` -- where do you get that from? This is another generated class but from the SAM client library.
```
> terra workspace list-users --format=json
[ {
  "role" : null,
  "members" : [ ]
}, {
  "role" : "READER",
  "members" : [ ]
}, {
  "role" : "WRITER",
  "members" : [ ]
}, {
  "role" : "OWNER",
  "members" : [ "mmdevverily@gmail.com" ]
} ]
```

Reply:
Ah, I was looking at the wrong class: https://github.com/DataBiosphere/terra-workspace-manager/blob/dev/src/main/java/bio/terra/workspace/service/iam/model/RoleBinding.java

Thanks for the output!

Reply:
Actually, this is a great point. Multiple client libraries have very similar model classes (e.g. RoleBinding, Status) and it would be great if we could share generated classes. I think this has come up for discussion before, but it was slightly complicated/unclear how to share snippets of the OpenAPI yaml specification that defines the generated classes.

Not sure if there's a ticket for this, but perhaps worth checking what the status of that was at standup today.